import-export-cli/integration/testutils/api_testUtils.go

```diff
@@ -274,6 +317,23 @@ func exportAPI(t *testing.T, name string, version string, provider string, env s
 	return output, err
 }
 
+func exportAPIRevision(t *testing.T, name string, version string, provider string, revision string, env string) (string, error) {
```

Comment:
```suggestion
func exportAPIRevision(t *testing.T, name, version, provider, revision, env string) (string, error) {
```
Shall we change like this to improve the readability and resuce the characters in a code line?

Reply:
Fixed with https://github.com/wso2/product-apim-tooling/pull/728/commits/0901d9dcfc9501ea1bac6682987f76eae977e297